crates/sui-graphql-rpc/src/config.rs

```diff
@@ -256,7 +259,15 @@ impl ServiceConfig {
         self.limits.request_timeout_ms
     }
 
-    /// Maximum length of a query payload string.
+    /// The maximum bytes allowed for the JSON object in the request body of a GraphQL mutation.
+    /// It is the value of the maximum transaction bytes (including the signatures) allowed by the
+    /// protocol, plus the Base64 overhead (4/3 of the original string), plus the max query
+    /// payload size allowed.
+    async fn max_mutation_payload_size(&self) -> u32 {
+        self.limits.max_mutation_payload_size
+    }
+
+    /// The maximum bytes allowed for the JSON object in the request body of a GraphQL read query.
```

Comment:
update this tomorrow to showcase clearly the difference between this one and mutation